test/github/client/github-client.test.ts

```diff
@@ -87,7 +87,7 @@ describe("GitHub Client", () => {
 		const pageSize = 5;
 		const page = 1;
 
-		githubAccessTokenNock(githubInstallationId, "installation token");
+		githubUserTokenNock(githubInstallationId, "installation token");
```

Comment:
I changed the function name because we also needed to create a global github app token nock, so I changed the name to make it more explicit which token we're getting.